docs/source/api_reference/regression.rst

```diff
@@ -159,6 +159,17 @@ Adapters to other interfaces
 
     SklearnProbaReg
 
+Adapters to other classifiers
+----------------------------
+
+.. currentmodule:: skpro.classification.sklearn_classifiers._sklearn_proba_class
```

Comment:
the result is a regressor, even if internally it uses a classifier, so it should be part of the `regression` module.

Similarly, the name of the estimator should not be "classifier", since it implies the estimator is a classifier.

I will think of a name, perhaps you can also try to come up with sth.